### PR TITLE
add lethargy w/ npm autoupdate

### LINKED ISSUE #9256: [Request] Add Lethargy
**Library name:*Lethargy* 
**Git repository url:**https://github.com/d4nyll/lethargy.git
**npm package url(optional):** 
**License(s):*ISC*
**Official homepage:**https://github.com/d4nyll/lethargy
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster than just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #9256 
@kennynaoh @x09326 Please help me review this pull request, thank you.
Manually add the other versions from git repo because npm auto-update can't get some of the old versions

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required):https://github.com/d4nyll/lethargy
 * Official website (optional, not the repository):
 * NPM package url (optional):https://www.npmjs.com/package/lethargy
 * GitHub / Bitbucket popularity (required):
   - Count of stars:96
   - Count of watchers:3
   - Count of forks:11
 * NPM download stats (optional):
   - Downloads in the last day:8
   - Downloads in the last week:89
   - Downloads in the last month:602

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #9256 , cc @d4nyll

### CODE REVIEW

**Comment by x09326** [comment]
LGTM

File: ajax/libs/lethargy/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "keywords": [
+    "scroll",
+    "wheel",
+    "mouse",
+    "mousewheel",
+    "scrolljacking",
+    "inertial",
+    "trackpad",
+    "magic",
+    "mouse"
```
Comment (kennynaoh) [inline ajax/libs/lethargy/package.json]: duplicated keywords

Comment (kennynaoh) [inline ajax/libs/lethargy/package.json]: dupilcated keyword `mouse`

File: ajax/libs/lethargy/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  ],
+  "author": "Daniel Li <dan@danyll.com>",
+  "license": "MIT",
+  "npmName": "lethargy",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "lethargy*.+(js|css|map)"
```
Comment (kennynaoh) [inline ajax/libs/lethargy/package.json]: why `css`?
Could `lethargy*` will be better?

Comment (pvnr0082t) [inline ajax/libs/lethargy/package.json]: Because you don't know if there will be `.css` file in the future. `lethargy*` will get something we don't need.

**Comment by pvnr0082t** [comment]
ping @kennynaoh @x09326

**Comment by x09326** [comment]
LGTM

**Comment by kennynaoh** [comment]
@pvnr0082t 
Is the title change to `add lethargy w/ npm autoupdate` better?
Thanks!

**Comment by pvnr0082t** [comment]
@kennynaoh fixed.

**Comment by kennynaoh** [comment]
LGTM
ping @PeterDaveHello

**Pull request merged**